src/routes/movies/CollectionsView.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { FC, useCallback } from 'react';
+
+import ViewItemsContainer from '../../components/common/ViewItemsContainer';
+
+interface CollectionsViewI {
```

Comment:
It looks like we could create a `LibraryView` (or similarly named) interface for all these views since they accept the same prop.

Reply:
https://github.com/jellyfin/jellyfin-web/pull/3792/commits/1c6b1fc4785168b159ae8c42bf89f5d7cc1ed248